src/Fabulous.XamarinForms/Views/_View.fs

```diff
@@ -62,6 +62,26 @@ module XFView =
                     view.TranslateTo(data.X, data.Y, data.AnimationDuration, data.Easing)
                     |> ignore)
 
+    let TranslateXTo =
```

Comment:
I just noticed Translate/Scale/Rotate are actually from VisualElement instead of View. Could you move all those to VisualElement.fs, please?
https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.visualelement?view=xamarin-forms

Reply:
Yeah you are right. We were duplicating some of the extensions :) . Moved to VisualElement and delete the duplicate ones and right API docs for it